resources/validations/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM openjdk:17-jdk-buster
```

Comment:
```suggestion
FROM openjdk:17-jdk-alpine
```

B: I did a comparison of the different container variants. Debian Buster and Alpine variants, and Debian with is stability guarantees does not disappoint. :-) Can we switch to Alpine Linux?

```sh
$ docker scan openjdk:17-jdk-buster
The Snyk version installed on your system does not match the docker scan requirements (>=1.385.0), using embedded Snyk version instead.

Testing openjdk:17-jdk-buster...

Organization:      ohsh6o
Package manager:   apk
Project name:      docker-image|openjdk
Docker image:      openjdk:17-jdk-buster
Platform:          linux/amd64
Licenses:          enabled

Tested 183 dependencies for known issues, found 160 issues.

$ docker scan openjdk:17-jdk-alpine
The Snyk version installed on your system does not match the docker scan requirements (>=1.385.0), using embedded Snyk version instead.

Testing openjdk:17-jdk-alpine...

Organization:      oohsh6o
Package manager:   apk
Project name:      docker-image|openjdk
Docker image:      openjdk:17-jdk-alpine
Platform:          linux/amd64
Licenses:          enabled

✓ Tested 20 dependencies for known issues, no vulnerable paths found.
```

Reply:
I am going to resolve this for now in favor of deciding it later in 18F/fedramp-automation#73 when we have a better idea of what GSA and/or TTS and/or FedRAMP specific security standards exist.